crates/web-plugins/didcomm-messaging/Cargo.toml

```diff
@@ -23,6 +23,7 @@ thiserror.workspace = true
 tokio = { workspace = true, features = ["full"] }
 hyper = { workspace = true, features = ["full"] }
 axum = { workspace = true, features = ["macros"] }
+chrono.workspace = true
```

Comment:
why have you added the `chrono` dep here?

Reply:
Yes, but by mistake. :upside_down_face: